modules/nw-sriov-configuring-operator.adoc

```diff
@@ -108,8 +108,7 @@ include::snippets/technology-preview.adoc[]
 [id="about-network-resource-injector_{context}"]
 == About the Network Resources Injector
 
-The Network Resources Injector is a Kubernetes Dynamic Admission Controller
-application. It provides the following capabilities:
+The Network Resources Injector is a Kubernetes Dynamic Admission Controller application, which provides the following capabilities:
```

Comment:
Verify that this should be a non-restrictive clause.

Reply:
I guess it was more a technical question. If it's supplementary, yup, non-restrictive clause set off by a comma. 👍